frontend/src/services/http/mock-data/service-docs.ts

```diff
@@ -0,0 +1,193 @@
+import { GetServiceDocResponse } from 'msadoc-client';
+
+const allServiceDocs: GetServiceDocResponse[] = [
```

Comment:
TODO: find a better way to read from file system in follow-up PR?

Reply:
We could import these from JSON files (it might be necessary to add a TSConfig setting for this to be possible). But I must say that the current solution actually looks pretty good to me. Especially because it is type-safe. Do you have any particular reason why you would prefer a different way to import the mock data?

Reply:
Well, they need manual updating... I'd rather somehow automate that using the examples in the `examples` directory..

Reply:
OK that makes sense 👍

In that case, we could indeed simply import the JSON using the following syntax:

```
import anExample from '(arbitrary-path)/notification.service.msadoc.json'
```


Not sure if it works, but maybe you can even import JSON from outside the root. So we could directly go into the examples folder:
```
import anExample from '../../../examples/notification.service.msadoc.json'
```

Alternatively, copy the examples into the frontend project and maybe add a CI action that checks if the JSON examples are up-to-date?